.gitignore

```diff
@@ -0,0 +1,162 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Can we also add *.swp?

Reply:
sure, but this file is complete overkill. this REPO should be very light weight. :)